oat-sa/extension-tao-testqti
# [TAO-9473] WTP-1466 - [ACT] [TTS - FE] Create TTS plugin

Related to : https://oat-sa.atlassian.net/browse/TAO-9475
 
Register new APIP Text To Speech plugin

#### How to test
 
- Go to `config/taoTests/test_runner_plugin_registry.conf.php` and change `active` to `true` for APIP Text To Speech plugin
- Prepare an instance of tao with taoAct extension
- Prepare a test with enabled APIP Text To Speech plugin
- Execute test as a test taker
- Make sure that APIP Text To Speech plugin is available in custom toolbox

Companion PR :
 - [ ] https://github.com/oat-sa/extension-tao-act/pull/1258
 - [x] https://github.com/oat-sa/tao-test-runner-qti-fe/pull/68
